### PR TITLE
Colon appears in imagename make exporttoxlsx script to fail

### PR DESCRIPTION
**Pull request by tcarmeli1**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-39253

## Description
colon appears in imagename make exporttoxlsx script to fail

### CODE REVIEW

**Review by jbabazadeh — APPROVED** [review]

File: Packs/EmailCommunication/ReleaseNotes/2_0_29.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### PreprocessEmail
+
+- Updated the preprocess script to handle attachments in the following format: <the image ID\>-imageName-<the image name\>.
```
Comment (ShirleyDenkberg) [inline Packs/EmailCommunication/ReleaseNotes/2_0_29.md:6]:
```suggestion
Updated the preprocess script to handle attachments in the following format: <the image ID\>-imageName-<the image name\>.
```

File: Packs/Gmail/ReleaseNotes/1_3_21.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Gmail
+
+- Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```
Comment (ShirleyDenkberg) [inline Packs/Gmail/ReleaseNotes/1_3_21.md:6]:
```suggestion
Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```

File: Packs/GmailSingleUser/ReleaseNotes/1_4_2.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Gmail Single User
+
+- Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```
Comment (ShirleyDenkberg) [inline Packs/GmailSingleUser/ReleaseNotes/1_4_2.md:6]:
```suggestion
Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_9.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_9.md:6]:
```suggestion
Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_4_3.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS O365
+
+- Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_4_3.md:6]:
```suggestion
Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_12.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Microsoft Graph Mail Single User
+
+- Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
+
+##### O365 Outlook Mail (Using Graph API)
+
+- Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_6_12.md:10]:
```suggestion
Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.

##### O365 Outlook Mail (Using Graph API)

Updated the attachments names to be in the following format: <the image ID\>-imageName-<the image name\>.
```

**Comment by ShirleyDenkberg** [comment]
@jbabazadeh Doc review completed.

**Comment by tcarmeli1** [comment]
Hi @yuvalbenshalom ,
Can you please assist with a force merge on this one?
2 tpbs failed and both have open tickets https://jira-dc.paloaltonetworks.com/browse/CIAC-9721, https://jira-dc.paloaltonetworks.com/browse/CRTX-122260.

**Pull request merged**